packages/storycrawler/src/browser/stories-browser.ts

```diff
@@ -80,6 +80,7 @@ export class StoriesBrowser extends BaseBrowser {
         timeout: 60_000,
       },
     );
+    await this.page.waitForTimeout(500);
```

Comment:
It seems to throw an `Execution context was destroyed` error if we don't wait for some time here, so I added some timeouts. I'm not sure why this is happening though.
ref: https://github.com/reg-viz/storycap/actions/runs/8354226734/job/22867216673#step:7:99